app/_hub/tomkerkhove/microsoft_azure/index.md

```diff
@@ -101,3 +101,31 @@ You can deploy one of the following Kong templates from the marketplace:
 ## Running Kong on Azure Container Instances
 
 For more information, see [Running Kong on Azure Container Instances](./../microsoft_azure_container_instances).
+
+## Storing Kong metadata in Microsoft Azure
+When running Kong you also need to store its metadata somewhere - Microsoft Azure can help with this as well!
```

Comment:
We don't use the term "metadata" in Kong's documentation - to align with our typical terminology, I suggest:

```suggestion
When running Kong you need a datastore - Microsoft Azure can help with this as well!
```

Reply:
Good point, updated :+1: